spec/models/meter_reading_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+RSpec.describe MeterReading, type: :model do
+  let!(:meter_reading) { create(:meter_reading) }
+  let(:value) { meter_reading.value }
```

Comment:
what s the point of this let if you still use meter_reading.value on line 16

Reply:
done